README.md

```diff
@@ -62,4 +62,11 @@ The Phyloreferencing Ontology uses as its base IRI `http://ontology.phyloref.org
 
 Files served from `http://ontology.phyloref.org` (i.e. those in the `docs/` directory) are only updated upon a new release of this ontology. Development versions of these ontologies can be found in this repository, and may be different from the files available at the canonical IRIs.
 
+Terms of reuse
+--------------
+
+If you use the Phyloreferencing Ontology or other parts of this work in your research, for now please cite this repository.
+
+To the extent possible under law, the Phyloreferencing Project investigators who associated CC0 with this work have waived all copyright and related or neighboring rights to the Phyloreferencing Ontology and all other ontologies maintained within this repository. Please see the file [LICENSE](./LICENSE), or <http://creativecommons.org/publicdomain/zero/1.0/>.
```

Comment:
Do we need to say "investigators who associated CC0 with this work"? I think "the Phyloreferencing Project investigators have waived ..." is clearer and less ambiguous.

Reply:
This language is taken from the [corresponding Creative Commons template](https://creativecommons.org/2011/04/15/plaintext-versions-of-creative-commons-licenses-and-cc0/). I have no legal knowledge that would allow me to argue what, if any, consequences there are to changing it in the way you describe.

There's a [slightly different template for using CC0 for software](https://wiki.creativecommons.org/wiki/CC0_FAQ#May_I_apply_CC0_to_computer_software.3F_If_so.2C_is_there_a_recommended_implementation.3F), which we could consider adapting here (by replacing _software_ with _work_, for example).